examples/01_plotting/plot_3d_map_to_surface_projection.py

```diff
@@ -27,11 +27,23 @@
 fsaverage = datasets.fetch_surf_fsaverage()
 
 ##############################################################################
-# Sample the 3D data around each node of the mesh
-# -----------------------------------------------
+# Use mesh curvature to display useful anatomical information
+# on inflated meshes
+#
+# Here, we load the curvature map of the hemisphere under study,
+# and define a surface map whose value for a given vertex is
+# 0.75 if the curvature is positive, 0.25 if the curvature is negative.
 
+import numpy as np
 from nilearn import surface
 
+curv_right = surface.load_surf_data(fsaverage.curv_right)
+curv_right_sign = (np.sign(curv_right) + 1) / 8 + 0.25
```

Comment:
```suggestion
curv_right_sign = (np.sign(curv_right) + 1) / 4 + 0.5
```

Reply:
Haha I think I actually need a mix of our two lines for values to be between .25 and .75, like so:
`curv_right_sign = (np.sign(curv_right) + 1) / 4 + 0.25`

```python
>>> a = np.array([-2, 0, 2])
>>> np.sign(a)
array([-1,  0,  1])
>>> (np.sign(a) + 1) / 8 + 0.25
array([0.25 , 0.375, 0.5  ])
>>> (np.sign(a) + 1) / 4 + 0.25
array([0.25, 0.5 , 0.75])
```

Is this what you expected?